tests/auth/managers/simple/routes/__init__.py

```diff
@@ -0,0 +1,16 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
It should be `router` and not `routes`

Reply:
I followed the same convention here even my mind ping me was that routers. I think all should be routers since we are creating a router in each router file :) 

https://github.com/apache/airflow/tree/main/tests/api_fastapi/core_api/routes
https://github.com/apache/airflow/tree/main/tests/api_fastapi/execution_api/routes

Reply:
I am happy to rename it `routes`, but then we should be consistent and do the same in `src`. I just want to use the same name in `src` and `tests`

Reply:
Amazing, renamed the src to routes to be consistent